services/headless-lms/migrations/20240312115600_add_suspected_cheaters.up.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE suspected_cheaters (
+  id UUID DEFAULT uuid_generate_v4() PRIMARY KEY,
+  student_id UUID NOT NULL REFERENCES course_module_completion,
```

Comment:
user_id, REFERENCES users

Reply:
please I cant recall, why do we need user_id when we already have student_i?